### PR TITLE
Added Cart, Wishlist utils & changes in product list

### PR DESCRIPTION
**Pull request by arulk8**
1. Added Wishlist and Cart UI.
2. Added add to wishlist and cart in product list view.
3. Updated Routes in Home page and Header.
4. need to work on wishlist and cart page

Deployment URL:
https://melodious-dieffenbachia-b37b79.netlify.app/

email: "adarshbalika@gmail.com",
password: "adarsh@1234",

### CODE REVIEW

**Review by raieegulhane — COMMENTED** [review]
Code looks fine. Please add preview.

**Comment by arulk8** [comment]
> Code looks fine. Please add preview.

Thanks for the review I have added the preview url

File: src/store/actions.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+            authorization: localStorage.getItem("authSession"),
+          },
+        }
+      );
+      console.log(cart);
+      dispatch({
+        type: actionTypes.ADD_TO_CART,
+        payload: cart,
+      });
+    } catch (err) {
```
Comment (sashaboi) [inline src/store/actions.js:148]: if a user isnt logged in , it should take the user to the login page, not do this add to cart empty payload? 
clicking on add to cart button seems broken unless logged in. just add a navigate link to your login page

**Review by sashaboi — APPROVED** [review]
Hello ! the home page images seem to have an issue loading do have a look at that. 
Other pages cant seem to get a hold of images as well , do check that out.

also could you add a login with dummy credentials , so a user might login easily?

**Pull request merged**